lib/menu.rb

```diff
@@ -0,0 +1,19 @@
+class Menu
+
+  def initialize
+    @menu = {
```

Comment:
It would be better for future to use to have option to initialise menu with different dishes so, I think its better to use constant here and use it as default value.